templates/example_qc_metric_eval.ipynb

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    sort_function = lambda row: sorted(re.findall(r'fov(\d+)|point(\d+)', row['fov'].lower())[0])[1]
</pre><p>We're going to have all kinds of weird naming schemes for the points, we won't be able to rely on this. We'll probably need to read the JSON that was created to figure out the right order. Can this be adapted so that given a list of the weird names in the correct order, it will order them accordingly? Can put a TODO here, and do that in the next PR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>@ngreenwald we could create a map of each FOV name to a numerical index and have the user refer to that. It would make things much simpler for us downstream too: instead of working with long string names we can instead use a numerical key to identify them instead.</p>